tests/unitTests/test.h

```diff
@@ -851,6 +1000,11 @@ class TestMooneyRivlin : public TestMaterialModel {
 
 // ----------------------------------------------------------------------------
 // Class for test of Holzapfel-Ogden material model
+// Implements two versions of the Holzapfel-Ogden model:
+// - Full anisotropic invariants (I1_bar, I4_f, I4_s, I8_fs) used by cardiac mechanics benchmark paper (Arostica et al., 2024)
+// - Modified anisotropic invariants (I1_bar, I4_bar_f, I4_bar_s, I8_bar_fs) used by svFSIplus
+// Also, the strain energy includes the Simo-Taylor 91 volumetric penalty term,
+// with parameter HolzapfelOgdenParams.kappa
 class TestHolzapfelOgden : public TestMaterialModel {
```

Comment:
In a perfect world, we would make `computeStrainEnergy` part of the (object-based) material models. Just to keep in mind for the future.

Reply:
Yes, is the class hierarchy I've done here what we had in mind for the object-oriented material model redesign?

Reply:
Yup!